content/kubernetes/release-notes/k8s-6-2-18-tbd.md

```diff
@@ -0,0 +1,140 @@
+---
+Title: Redis Enterprise for Kubernetes release notes 6.2.18-TBD (Nov 2022)
+linktitle: 6.2.18-TBD (Nov 2022)
+description: Support added for additional distributions, as well as feature improvements and bug fixes. 
+weight: 60
+alwaysopen: false
+categories: ["Platforms"]
+aliases: [
+    /kubernetes/release-notes/k8s-redis-on-flash.md,
+    /kubernetes/release-notes/k8s-redis-on-flash/, 
+    /kubernetes/release-notes/k8s-6-2-18-tbd.md,
+    /kubernetes/release-notes/k8s-6-2-18-tbd/,   ]
+---
+## Overview
+
+The Redis Enterprise K8s 6.2.18-tbd supports the Redis Enterprise Software release 6.2.18 and includes feature improvements and bug fixes.
+
+The key bug fixes, new features, and known limitations are described below.
+
+## Images
+
+This release includes the following container images:
+
+* **Redis Enterprise**: `redislabs/redis:6.2.18-49` or  `redislabs/redis:6.2.18-49.rhel8-openshift`
+* **Operator**: `redislabs/operator:6.2.18-tbd`
+* **Services Rigger**: `redislabs/k8s-controller:6.2.18-tbd` or `redislabs/services-manager:6.2.18-tbd` (on the Red Hat registry)
+
+## New features
+
+* Support for [Redis on Flash]({{<relref "/kubernetes/re-clusters/redis-on-flash.md">}}) (RoF) (RED-78613)
+
+## Feature improvements
+
+* The `podSecurityPolicyName` field in `RedisEnterpriseCluster` resources is now deprecated for Kubernetes versions 1.24 or earlier, and invalid for 1.25 and later. Customers are advised to switch to using the [`PodSecurityAdmission`](https://kubernetes.io/docs/concepts/security/pod-security-admission/) or alternative methods to enforce pod security (RED-81921).
+
+* Added support for VMware [Tanzu Kubernetes Grid (TKG)](https://docs.vmware.com/en/VMware-Tanzu-Kubernetes-Grid/index.html), in addition to [Tanzu Kubernetes Grid Integration Edition (TKGI)](https://docs.vmware.com/en/VMware-Tanzu-Kubernetes-Grid-Integrated-Edition/index.html) that was previously and is still supported (RED-65630).
+
+* Added support for PEM encryption through the [Redis Enterprise cluster API](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/redis_enterprise_cluster_api.md)(RED-78613).
+
+* Hardened security context constraints to align with standards for OpenShift 4.11 (RED-83215).
+
+* Changed log collector default to avoid collection of non-Redis Enterprise logs and items (RED-83216).
+
+* Allowed configuration of the Redis Enterprise cluster (REC) service type (RED-84644).
+
+## Bug fixes
+
+* Allowed nginx ingress class annotation to contain "`ngnix`" as a substring instead of being exactly "`ngnix`" (RED-79205).
```

Comment:
We actually don't require anything now from the ingress class annotation. I'd rephrase this to something along these lines:
> Allow any ingress class name annotation when using NGINX ingress controller - this is no longer required to be exactly `nginx`.

Btw, this wasn't really a bug, maybe a limitation but working as designed. So, this is borderline between feature improvement and bug fix. Up to you :-)